CONTRIBUTING.md

```diff
@@ -98,9 +98,9 @@ vcpkg install ensmallen:x64-windows
 
 ### Build and Test
 
-This section describes how to build the **ensmallen** tests from source. **ensmallen** uses CMake as its build system and [Catch2](https://github.com/catchorg/Catch2) as the unit test framework.
+This section describes how to build the **ensmallen** tests from the source. **ensmallen** uses CMake as its build system and [Catch2](https://github.com/catchorg/Catch2) as the unit test framework.
 
-First, clone the source code from Github and change into the cloned directory. Or alternatively, you can download the latest relese from the [website](http://ensmallen.org) and extract it.
+First, clone the source code from Github and change it into the cloned directory. Alternatively, you can download the latest release from the [website](http://ensmallen.org) and extract it.
```

Comment:
I think that "change into so-and-so directory" is reasonably understood, and I'd strongly recommend against adding "it" to this sentence.
```suggestion
First, clone the source code from Github and change into the cloned directory. Alternatively, you can download the latest release from the [website](http://ensmallen.org) and extract it.
```